NetworkFactory.m

```diff
@@ -1,19 +1,62 @@
 classdef NetworkFactory
-    % A factory to initialise the network to be used within an ARTwarp run.
+    % Summary of this class goes here
     %   Detailed explanation goes here
     
     methods (Static)
-        function net = new_network(contours)
-            % Creates an empty 
-            weights = ones(max([contours.length]), 0);
-            net = Network(weights);
+        function net = new_network()
```

Comment:
@michaelstanway the tests now failed because they expect an argument given to `new_network`. Do you know how to fix this? We were puzzled that `new_network` does not require any arguments in your version.